### PR TITLE
add latest release

### LINKED ISSUE #341: How can I get Latest Release of Repository?

### PR DESCRIPTION
**Pull request by kamontat**
I already add the latest release, so please add to master and update version so that I can use in my project
Fix #341 

PS  I don't know how to push my fixed module to the maven server so, please. @emlagowski 

Thank you

### CODE REVIEW

**Comment by Shredder121** [comment]
It seems the indentation is off though.
And also, would a test be good for this feature?
(I'm not a member, so don't take my word on this)

**Comment by emlagowski** [comment]
@kamontat I'm also not a member.

@Shredder121 taking the opportunity, do you know how to run tests locally if I have no access to repositories and organisations that are referenced from the unit tests?

**Comment by KostyaSha** [comment]
You can read test code, it uses file with local configuration. Create additional account and run using it.

File: src/main/java/org/kohsuke/github/GHRepository.java [hunk 1]
```diff
@@ -298,6 +298,10 @@ public GHRef createRef(String name, String sha) throws IOException {
     public List<GHRelease> getReleases() throws IOException {
         return listReleases().asList();
     }
+    
+	public GHRelease latestRelease() throws IOException {
```
Comment (KostyaSha) [inline src/main/java/org/kohsuke/github/GHRepository.java]: should be `getLatestRelease`,
fix indentation

**Comment by emlagowski** [comment]
@KostyaSha I know that, but it still fails. Is there any list of what scopes an access token needs to pass the tests?

File: src/main/java/org/kohsuke/github/GHRepository.java (hunk 1 above)
Comment (kamontat) [inline src/main/java/org/kohsuke/github/GHRepository.java]: fixed, Thank you

- https://github.com/kohsuke/github-api/pull/343/commits/fd859815b039a84c3dd53e38ff8d41fefc2c2b23?diff=unified

**Comment by kamontat** [comment]
@Shredder121 I already testing on it in my own repository, so I sure it's work

but I don't push my test up because I test by using my own repository.

**Pull request merged**